src/main/java/com/jalasoft/jfc/model/audio/AudioConverter.java

```diff
@@ -38,6 +62,75 @@
      */
     @Override
     public FileResponse convert(Param param) throws ConvertException, CommandValueException, IOException, ZipJfcException {
-        return null;
+        if (param == null) {
+            throw new ConvertException(ErrorMessageJfc.AUDIO_CONVERT_ERROR_MESSAGE.getErrorMessageJfc(), this.getClass().getName());
+        }
+
+        AudioParam audioParam = (AudioParam) param;
+        FileResponse fileResponse = new FileResponse();
+
+        String commandString;
+
+        generateAudio(audioParam);
+
+        ContextStrategy commandAudioContext = new ContextStrategy(commandAudioList);
+
+        try {
+            commandString = commandAudioContext.buildCommand();
+
+            Process audioConvertProcess = Runtime.getRuntime().exec(commandString);
+            audioConvertProcess.waitFor();
+
+            if (param.isMetadata()) {
+                MetadataConverter metadataConverter = new MetadataConverter();
+                metadataConverter.convert(param);
+            }
+
+            zipFile(audioParam);
+
+            FolderRemover.removeFolder(audioParam.getOutputPathFile() + audioParam.getFolderName());
+
+            fileResponse.setName(audioParam.getOutputName());
+            fileResponse.setStatus(MessageResponse.SUCCESS200.getMessageResponse());
+            fileResponse.setDownload(zipPath);
+        } catch (Exception e) {
```

Comment:
"Exception" must be someone of our exceptions

Reply:
This code needs that exception "audioConvertProcess.waitFor();"